### ISSUE TITLE
Wrong Learning Rate in superpoint+lightglue_megadepth.yaml

### ISSUE THREAD

**Issue by NoahYildiz** [cvg/glue-factory#161]
In gluefactory/configs/superpoint+lightglue_megadepth.yaml there is the wrong learning rate initially set. 
The original Paper said: 

> "Weights are initialized from the pretrained model on homographies, Training starts with a learning rate of 1e-5 and we exponentially decay it by 0.95 in each epoch after 10 epochs, and stop training after 50 epochs [...]"

In the config it starts with 1-4 and starts decaying after 30 epochs the same way it did in the pretraining.

I don't know if it is intended to be different from the paper but i think it is a little bit confusing if it's different.

Thanks!

**Comment by Phil26AT** [comment]
Hey, indeed there is a slight inconsistency, mostly because we retrained LG with similar setups for more models after release. Issues #43 also raised this. We are sorry for this inconsistency.

I suggest to stick with the learning rate in the codebase. We also found that a learning rate of 2e-4 works slightly better.
